src/schema/posts.ts

```diff
@@ -1367,7 +1367,10 @@ export const resolvers: IResolvers<any, Context> = {
           { id },
           {
             deleted: true,
-            flags: updateFlagsStatement<Post>({ deleted: true }),
+            flags: updateFlagsStatement<Post>({
+              deleted: true,
+              deletedBy: ctx.userId,
+            }),
```

Comment:
Think it's a good thing for historical purposes

Reply:
Agreed